index.js

```diff
@@ -78,14 +78,15 @@ function fastifyOauth2 (fastify, options, next) {
   const startRedirectPath = options.startRedirectPath
   const tags = options.tags || []
   const schema = options.schema || { tags }
+  const cookieOpts = options.cookie || {
```

Comment:
Can we have something like
```suggestion
  const cookieOpts = Object.assign({}, options.cookie, {
```

Reply:
Sure|